### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by raysmithdev**
@simpixelated here you go. FYI: just make sure you are running npm v3.

### CODE REVIEW

File: src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+      return photo.checked !== true
+    })
+
+    this.showAlert('Image(s) have been deleted')
+
+    this.setState({
+      photos,
+      deleteItemsModalOpen: false
+    })
+  }
```
Comment (simpixelated) [inline src/components/App.jsx:99]: The naming of these functions is a bit hard to parse. What's the difference between `handleDelete` and `deleteItem`? Different names and / or some comments in the code would help.

Also, is there any way you could reduce some duplicate code by combining the single and multiple delete functions?

File: src/components/PhotoGallery.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+              key={index}
+              titleBackground={'#ccc'}
+           >
+            <a href={photo.userImageURL} target="_blank">
+            <img src={photo.userImageURL} />
+            </a>
+          </GridTile>
+        )
+      })
+  }
```
Comment (simpixelated) [inline src/components/PhotoGallery.jsx:60]: It looks like the `GridTitle.title` is the only difference between the `props.editable` conditional. If that's the case, could you DRY this up a bit by conditionally setting a different `title` value?

File: src/components/PhotoGallery.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+const PhotoGallery = (props) => {
+  return (
+    <div>
+      <Sortable
+          onChange={(order, sortable) => { props.onChange(order) }}
+          className="container"
+      >
+        {renderContent(props)}
```
Comment (simpixelated) [inline src/components/PhotoGallery.jsx:70]: Any reason not to make this a separate component instead of a function?

File: src/components/App.jsx (hunk 1 above)
Comment (raysmithdev) [inline src/components/App.jsx:99]: I just wanted to split them up for clarity and make the functions slim but I can definetly put them into one function.

File: src/components/PhotoGallery.jsx (hunk 2 above)
Comment (raysmithdev) [inline src/components/PhotoGallery.jsx:60]: Yes I could actually use like condition : true ? false inside the title prop perhaps

File: src/components/PhotoGallery.jsx (hunk 3 above)
Comment (raysmithdev) [inline src/components/PhotoGallery.jsx:70]: i did split it up originally but felt I was having too much components for the objective, after reading article on medium however I now know that you can never have too many components

Comment (simpixelated) [inline src/components/PhotoGallery.jsx:70]: Yeah I think modularity and granularity is a good thing when it comes to components.
